navigation/router.js

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+import { SafeAreaView } from "react-navigation";
+import { createStackNavigator } from "react-navigation-stack";
+import {
+  createMaterialTopTabNavigator,
+  MaterialTopTabBar,
+} from "react-navigation-tabs";
+import LoginScreen from "../screens/LoginScreen";
+import RegisterScreen from "../screens/RegisterScreen";
+import { Ionicons } from "@expo/vector-icons";
+
+const AuthNavigator = createMaterialTopTabNavigator(
+  {
+    Login: {
+      screen: LoginScreen,
+      navigationOptions: {
+        tabBarLabel: "Login",
+        showIcon: true,
+        tabBarIcon: () => {
+          return <Ionicons name="md-contact" size={20} color={"black"} />;
+        },
+      },
+    },
+    Register: {
+      screen: RegisterScreen,
+      navigationOptions: {
+        tabBarLabel: "Register",
+        showIcon: true,
+        tabBarIcon: () => {
+          return <Ionicons name="md-create" size={20} color={"black"} />;
+        },
+      },
+    },
+  },
+  {
+    initialRouteName: "Login",
+    tabBarComponent: props => (
+      <SafeAreaView>
```

Comment:
@KonradSobolewski I fixed your issue with overlapping notch, but don't you think that it would look better at the bottom of the screen? I actually don't know, we need to make a decision, but also have to keep in mind that tab navigations on other screens (with tv shows) must have the same position as this one to keep it consistent.